app/views/videos/show/_translate_frame.html.erb

```diff
@@ -19,7 +19,7 @@
       <div role="tabpanel">
         <div class="tab-content">
           <div role="tabpanel" class="tab-pane fade in active tab-amara" id="tab-amara">
-            <span>If you'd like to use Amara's translation interface, you can use the <b><%= link_to 'link to the Amara video', video.amara_link, :target => '_amara' %></b>. To earn points, submit your published Amara link below.</span>
+            <span>If you'd like to use Amara's translation interface, you can use the <b><%= link_to 'link to the Amara video', video.amara_link, :target => '_amara' %></b>. To earn points, submit your published Amara link below. You can find this link by clicking <kbd>Burmese</kbd> in the languages panel.</span>
```

Comment:
Customer said something about removing the first phrase, "If you'd like to use Amara's translation interface", because that tab assumes you do want to use their interface.